roles/deploy/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 rollback_path: "{{ app_path }}/rollback"
 rollback_sql_path: "{{ app_path }}/rollback.sql.gz"
+rollback_tmp_path: "/var/lib/postgresql/rollback.sql.gz"
```

Comment:
This is much better. But it's still a world-readable directory. Even though the file is only there for a short time, you can watch the directory with inotify and read the file as soon as it's created.

What do you think about using `mktemp` which creates a private file. `root` will still be able to move it. :thinking: Looking at our current setup, `/home/openfoodnetwork` is actually world-readable as well. So you are not making anything worse here.

Reply:
Maybe we can just open a separate issue and deal with it in a separate PR? your call